content/pt-br/docs/reference/glossary/container-runtime-interface.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Container runtime interface (CRI)
+id: cri
+date: 2019-03-07
+full_link: /docs/concepts/overview/components/#container-runtime
+short_description: >
+    Uma API para agentes de execução de contêineres se integrarem com o kubelet
+
+
+aka:
+tags:
+- fundamental
+---
+A interface de execução de contêiner (CRI) é uma API para agentes de execução de
+contêineres se integrarem com o kubelet em um nó.
+<!--more-->
+
+Para maiores informações, veja as APIs e especificações do [CRI](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/container-runtime-interface.md).
```

Comment:
This file appears very outdated when compared with the current English version. Could you please review this translation?